CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+https://www.conventionalcommits.org
```

Comment:
Can you give some context here?
Or make a new Todo in Trello that we add some context later..